Ginger/Ginger/ValueExpression/ValueExpressionEditorPage.xaml.cs

```diff
@@ -537,20 +643,20 @@ private void AddVariables()
             {
                 TreeViewItem tviVars = new TreeViewItem();
                 tviVars.Items.IsLiveSorting = true;
-                SetItemView(tviVars, GingerDicser.GetTermResValue(eTermResKey.BusinessFlow) + " " + GingerDicser.GetTermResValue(eTermResKey.Variables), "", "@Variable_16x16.png");
-                xObjectsTreeView.Items.Add(tviVars);
+                SetItemView(tviVars, GingerDicser.GetTermResValue(eTermResKey.BusinessFlow) + " " + GingerDicser.GetTermResValue(eTermResKey.Variables), "", eImageType.Variable);
+                child.Items.Add(tviVars);
 
                 foreach (VariableBase v in mContext.BusinessFlow.Variables.OrderBy("Name"))
                     InsertNewVarTreeItem(tviVars, v);
                 InsertAddNewVarTreeItem(tviVars, eVariablesLevel.BusinessFlow);
-                tviVars.IsExpanded = true;
+                //tviVars.IsExpanded = true;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6356765)